### PR TITLE
Consistently use Java style for array declarations

### PR DESCRIPTION
**Pull request by UniQP**

### CODE REVIEW

File: mapswithme-api/src/com/mapswithme/maps/api/MapsWithMeApi.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
   public static void showMapsWithMeUrl(Activity caller, PendingIntent pendingIntent, double zoomLevel, String url)
   {
     final Uri uri = Uri.parse(url);
-    final String latlon[] = uri.getQueryParameter("ll").split(",");
+    final String[] latlon = uri.getQueryParameter("ll").split(",");
```
Comment (rsudev) [inline mapswithme-api/src/com/mapswithme/maps/api/MapsWithMeApi.java]: Not sure, but I think this is a file provided by Mapswithme. In that case we shouldn't modify it, I'd say.

Comment (UniQP) [reply mapswithme-api/src/com/mapswithme/maps/api/MapsWithMeApi.java]: I also created a PR for Mapswithme: https://github.com/mapsme/api-android/pull/34.

But you're right. We should fix this occurrence by upgrading the the Mapswithme API after the mentioned PR is merged.

Comment (Lineflyer) [reply mapswithme-api/src/com/mapswithme/maps/api/MapsWithMeApi.java]: Is there any update available for this API? 
Its not working in cgeo anyway at the moment...see related issue.

Comment (UniQP) [reply mapswithme-api/src/com/mapswithme/maps/api/MapsWithMeApi.java]: Well, they have a git repository for the API.

Comment (Lineflyer) [reply mapswithme-api/src/com/mapswithme/maps/api/MapsWithMeApi.java]: Seems untouched since several years AFAICS.

**Pull request merged**